application/controllers/RoleController.php

```diff
@@ -244,11 +246,14 @@ public function auditAction()
             ]
         ));
 
-        $this->addControl($header);
```

Comment:
It's not an option to remove the header from the controls always. It must not scroll away.

You'd need to do that conditionally instead, though that's not really an option as this can only be known at this stage by accessing the `format` URL parameter, which I don't like.

I'd suggest to set what's inside `$header` also as page title. (`$this->setTitle()`) This way it shows naturally up on the export (on every page) and it even has a benefit for normal views in the browser.

Reply:
sorry didn't see the "scroll away" part, done